examples/iam_policy.json

```diff
@@ -2,40 +2,50 @@
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "CanListObjectsInAthenaBuckets",
+            "Sid": "AllowListAllMyBuckets",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetBucketLocation",
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": [
+                "*"
+            ]
+        },
+        {
+            "Sid": "AllowListBucket",
             "Effect": "Allow",
             "Action": [
                 "s3:ListBucket"
             ],
             "Resource": [
-                "arn:aws:s3:::aws-athena-query-results-*",
+                "arn:aws:s3:::moj-analytics-lookup-tables",
                 "arn:aws:s3:::alpha-athena-query-dump"
             ]
         },
         {
-            "Sid": "CanWriteToDefaultAthenaBucket",
+            "Sid": "AllowGetObject",
             "Effect": "Allow",
             "Action": [
-                "s3:GetObject",
-                "s3:PutObject"
+                "s3:GetObject"
             ],
             "Resource": [
-                "arn:aws:s3:::aws-athena-query-results-*"
+                "arn:aws:s3:::moj-analytics-lookup-tables/*"
             ]
         },
         {
-            "Sid": "CanReadAthenaExamples",
+            "Sid": "AllowGetPutObject",
             "Effect": "Allow",
             "Action": [
                 "s3:GetObject",
-                "s3:ListBucket"
+                "s3:PutObject"
             ],
             "Resource": [
-                "arn:aws:s3:::athena-examples*"
+                "arn:aws:s3:::aws-athena-query-results-593291632749-eu-west-1/*"
```

Comment:
Not sure we want to list the region and account no? Think `aws-athena-query-results-` is better?

Reply:
Yep, that makes more sense on reflection. I'll revert it back.